TM1py/Services/CubeService.py

```diff
@@ -84,6 +84,15 @@ def update(self, cube):
         request = "/api/v1/Cubes('{}')".format(cube.name)
         return self._rest.PATCH(request, cube.body)
 
+    def check_rules(self, cube):
+        """ Check rules syntax for existing cube on TM1 Server
+
+        :param cube: instance of TM1py.Cube
+        :return: response
+        """
+        request = "/api/v1/Cubes('{}')/tm1.CheckRules".format(cube.name)
+        return self._rest.POST(request)
```

Comment:
For this method you are passing an instance of the `Cube` class, but actually we are just making use of the name of the cube.
I suggest we change the argument to `cube_name` (string) or we check whether the argument is of type `Cube` or `str` inside the method